CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+## Contributing to Sparkles App
+
+Thank you for considering contributing to the Sparkles App! We appreciate your help in making this application better. Please follow the guidelines below to get started.
+
+### How to Contribute
+
+#### 1. Fork the Repository
+To contribute to this project, you'll need to create a personal copy of the repository. You can do this by clicking the "Fork" button in the upper right corner of the repository page.
+
+#### 2. Clone Your Fork
+Once you've forked the repository, clone it to your local machine:
+
+```bash
+git clone https://github.com/SparklesIDE/Sparkles-app.git
+cd Sparkles-app
+```
+
+#### 3. Create a New Branch (Optional)
+Create a new branch for your feature or bug fix:
+git checkout -b feature/your-feature-name
```

Comment:
wouldn't it be git branch -M your-feature-name?

Reply:
It is to create a new one and change branch at the same time.

Great for isolating the original branch. Your suggestion is to rename existing branch. I do not recommend it.

https://git-scm.com/docs/git-checkout/pt_BR

Reply:
ok